cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -570,7 +570,10 @@ liveViewPostSetup lvbe ncli nc = do
     nodeId =
       case nodeMode ncli of
         MockProtocolMode -> namenum (ncNodeId nc)
-        RealProtocolMode -> Text.pack $ "Port: " <> show (naPort <$> nodeAddr ncli)
+        RealProtocolMode -> Text.pack $ "Port: " <>
+          case naPort <$> nodeAddr ncli of
+            Nothing -> "-"
```

Comment:
Is this what we want to display in the `Nothing` case? Or is there another means by which we can identify the node?

Reply:
> Or is there another means by which we can identify the node?

I don't know it.